packages/pluggableWidgets/tree-view-web/src/components/hooks/treeViewAccessibility.tsx

```diff
@@ -0,0 +1,136 @@
+import { EventHandler, SyntheticEvent, useCallback, useMemo } from "react";
+import { TreeViewState } from "../TreeView";
+import { KeyboardHandlerHook, useKeyboardHandler } from "./useKeyboardHandler";
+
+export const enum FocusTargetChange {
+    FIRST = "FIRST",
+    LAST = "LAST",
+    PREVIOUS = "PREVIOUS",
+    NEXT = "NEXT"
+}
+
+export type TreeViewFocusChangeHandler = (
+    targetElement: EventTarget | null,
+    focusTargetChange: FocusTargetChange,
+    traverseOption?: "HORIZONTAL" | "VERTICAL"
+) => void;
+
+export const useTreeViewFocusChangeHandler = (): TreeViewFocusChangeHandler => {
+    return useCallback((targetElement, focusTargetChange, traverseOption) => {
+        if (targetElement && targetElement instanceof Element) {
+            const getTreeViewHeadersInElement = (element: Element | Document | null): HTMLHeadElement[] =>
+                element ? Array.from(element.querySelectorAll("li.widget-tree-view-branch")) : [];
```

Comment:
`querySelectorAll` always return an array :)
![image](https://user-images.githubusercontent.com/45102481/129767767-33f21036-640e-4e68-9598-063fa1c2cf85.png)

Reply:
It returns [a NodeList](https://developer.mozilla.org/en-US/docs/Web/API/NodeList), which has some utility functions but those only create an iterator. It's not really an array, so I would have to convert to an array no matter what.

Reply:
`.forEach` works 😅 , but as I said, is not a blocker. So i am going to resolve it

Reply:
yeah I just saw how you did it in other PR. But I need the filter and find functions 😄